### PR TITLE
Metrics query

### PR DESCRIPTION
**Pull request by DavidBSteele**
Added both metrics queries which were used at month end. We need to keep these in source control and improve them for future months.

### CODE REVIEW

File: src/SFA.DAS.Payments.sln [hunk 1]
```diff
@@ -264,11 +264,16 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.Payments.Monitoring
 EndProject
 Project("{00D1A9C2-B5F0-4AF3-8072-F6C62B433612}") = "SFA.DAS.Payments.Database.Legacy.Payments", "SFA.DAS.Payments.Database.Legacy.Payments\SFA.DAS.Payments.Database.Legacy.Payments.sqlproj", "{F8DC3227-6A79-44D5-89D8-6ACE0B8FCB37}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.Payments.ProviderPayments.LegacyExportService", "SFA.DAS.Payments.ProviderPayments.LegacyExportService\SFA.DAS.Payments.ProviderPayments.LegacyExportService.csproj", "{2CFF45F6-6398-4F3A-B3AE-03BC82F06F63}"
```
Comment (samuel-dev-dba) [inline src/SFA.DAS.Payments.sln]: why is this changed

File: src/SFA.DAS.Payments.sln [hunk 2]
```diff
@@ -264,11 +264,16 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.Payments.Monitoring
...
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.Payments.ProviderPayments.LegacyExportService", "SFA.DAS.Payments.ProviderPayments.LegacyExportService\SFA.DAS.Payments.ProviderPayments.LegacyExportService.csproj", "{2CFF45F6-6398-4F3A-B3AE-03BC82F06F63}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.Payments.ProviderPayments.LegacyExportService", "SFA.DAS.Payments.ProviderPayments.LegacyExportService\SFA.DAS.Payments.ProviderPayments.LegacyExportService.csproj", "{2CFF45F6-6398-4F3A-B3AE-03BC82F06F63}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.Payments.Monitoring.Jobs.JobService.Interfaces", "SFA.DAS.Payments.Monitoring.Jobs.JobService.Interfaces\SFA.DAS.Payments.Monitoring.Jobs.JobService.Interfaces.csproj", "{AED49A8C-EC1F-4649-B0FE-D0DB1DD07873}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.Payments.FundingSource.LevyFundedProxyService.UnitTests", "SFA.DAS.Payments.FundingSource.LevyFundedProxyService.UnitTests\SFA.DAS.Payments.FundingSource.LevyFundedProxyService.UnitTests.csproj", "{539FDC08-236E-4570-B33A-03836EC6B26A}"
```
Comment (samuel-dev-dba) [inline src/SFA.DAS.Payments.sln]: Why is this changed

File: src/SFA.DAS.Payments.sln (hunk 1 above)
Comment (DavidBSteele) [reply src/SFA.DAS.Payments.sln]: The only change I've made is to add the new solution folder. 

It looks like VS has created a new guid for 2 projects. I haven't made any explicit changes to them.

Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.sln]: Ok them looks fine to me
